Update: flutter getting started to use amplify studio. (#3973)
aws-amplify/docs: src/fragments/start/getting-started/flutter/generate-model.mdx

## Create data model

> 1. Navigate to the [Amplify Studio Quick Start](https://sandbox.amplifyapp.com/getting-started) and, choose to build a **To-do list** with **Flutter**.

Reviewer: The comma between `and` and `choose` is no longer necessary.

```suggestion
1. Navigate to the [Amplify Studio Quick Start](https://sandbox.amplifyapp.com/getting-started) and choose to build a **To-do list** with **Flutter**.
```

## Create data model

> 1. Navigate to the [Data feature in AWS Amplify Studio](https://sandbox.amplifyapp.com/getting-started) and, with **To-do list schema** selected, click **Create new schema**.

Reviewer: +1

> 1. Navigate to the [Data feature in AWS Amplify Studio](https://sandbox.amplifyapp.com/getting-started) and, with **To-do list schema** selected, click **Create new schema**.

Author: Thanks Chris. **Amplify Studio Quick Start** would be the correct page name.

    ![Create new schema](/images/lib/getting-started/flutter/generate-model-admin-ui-create.png)

> 

Reviewer 2: Further down on this page, under the **Generate the models locally** section, I think some of this may be outdated as well now that Amplify Studio interface changed:
```
On the Test locally in your app screen, you will be prompted with on-screen instructions
    1. **What type of app are you building?**
        1. Platform: **Cross-platform**
        2. Framework / Language: **Flutter**
        3. Click **Next**
    2. **(Optional) Create a new app**
        1. You should have already created a new Flutter app in a previous step, so go ahead and click **Next**
    3. **Install Amplify CLI to pull the data model**
        1. You should have already installed the Amplify CLI in a previous step
        2. Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
```
This is no longer accurate. Instead, I think the new steps allow you to essentially skip to the copy command. I suggest we update it to something like:
```
2. On the Local setup instructions screen, you will should see some on-screen instructions
    1. **Local setup instructions for** Flutter (This should already be selected for you)
    2. **Install Amplify CLI to pull the data model**
        1. You should have already installed the Amplify CLI in a previous step
        2. Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
```

Author: This section could definitely be made much simpler.  I have update it to reflect current Amplify Studio Steps.

## Generate the models locally

1. Click **Next: Test locally in your app** on the Data modeling screen to proceed
2. On the local setup instructions page, you will be prompted with on-screen instructions to pull the data model.
    1. **Install Amplify CLI to pull the data model**
        - You should have already installed the Amplify CLI in a previous step
        - Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
            
            ```bash
            amplify pull --sandboxId <UUID>

>             ```        

Reviewer 2: Nit: Trim whitespace?

Author: Good catch. removed the extra whitespace.

        - Make a note of the sandbox id, just in case!

>     2. As the rest of the steps will be covered by the tutorial, you can skip the remaining steps. 
> 3. Click **Deploy to AWS**

Reviewer 2: ```suggestion
    2. As the rest of the steps will be covered by the tutorial, you can skip the remaining steps and just click **Deploy to AWS** on the top right
```

>     2. As the rest of the steps will be covered by the tutorial, you can skip steps **2** and **3** by clicking **Next**. 

Reviewer 2: I think calling out steps 2 and 3 directly might be confusing seeing as how *this* list also has steps 2 and 3. It may be preferable to keep the original wording regarding skipping remaining steps?

Author: Update it to "skip the remaining steps".

> 3. Finally, Click **Deploy to AWS** at the end of step **3**.

Reviewer 2: Nit: Probably shouldn't use the word "Finally" when there's still another step to complete.

Also, I'm a little confused by what "at the end of step 3" means - do we even need this? Can we just have "Click Deploy to AWS" for this step?

Author: Yes, we shouldn't use Finally here. I have removed it and simplified step 3 to "Click **Deploy to AWS**".

> 4. In order to deploy a backend, you will need an AWS account. We will go over this in a later step when we’re ready to sync our app to the cloud. For now, keep the link to this page handy!

Reviewer 2: ```suggestion
3. In order to deploy a backend, you will need an AWS account. We will go over this in a later step when we’re ready to sync our app to the cloud. For now, keep the link to this page handy!
```
